app/assets/javascripts/adminComments.js

```diff
@@ -0,0 +1,75 @@
+var adminComments = (function() {
```

Comment:
This is used in another javascript library, but I guess Hound doesn't catch that.